site/server/app/com/fortysevendeg/exercises/app.scala

```diff
@@ -1,27 +1,12 @@
 package com.fortysevendeg.exercises
 
-import cats.Monad
 import cats.data.Coproduct
-import cats.free.Free
+
 import com.fortysevendeg.shared.free.ExerciseOp
 import com.fortysevendeg.exercises.services.free._
-import cats.~>
-import scalaz.concurrent.Task
-import com.fortysevendeg.exercises.services.interpreters.ProdInterpreters._
-import scalaz.\/
 
 object app {
-
-  type ExercisesApp[A] = Coproduct[ExerciseOp, UserOp, A]
-
-  implicit val interpreters: ExercisesApp ~> Task = exerciseOpsInterpreter or userOpsInterpreter
-
-  implicit class FreeOps[A](f: Free[ExercisesApp, A]) {
-
-    /** Run this Free structure folding it's ops to an effect capturing task */
-    def runTask: Throwable \/ A =
-      f.foldMap(interpreters).unsafePerformSyncAttempt
-
-  }
-
+  type ExercisesAndUserOps[A] = Coproduct[ExerciseOp, UserOp, A]
```

Comment:
I mentioned something similar in the PR from @juanpedromoreno. I don't believe the intermediate `Coproduct` aliases need to be named like these because they are not going to be used in isolation or composed in other context. Since they just represent aliases in the `Coproduct` tree constructions they may as well be named something like `C01`. If you disagree is fine, just a thought, since the only thing I need to know about those type aliases is in which order they are getting composed. It's similar to the use of the lambda name in type lambdas, this is just Scala being annoying in the lack of support for currying type constructors.